### PR TITLE
Mambo 34 remember me

### PR DESCRIPTION
**Pull request by jcger**

### CODE REVIEW

File: src/lib/resource/interceptor/refresh.es [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    console.log('retry')
+    options = {
+      headers: {
+        'Authorization': 'Basic ' + window.btoa('dalmeida:areas')
+      }
+    }
+    return retryReq.request(method, options)
+  }
+
+  onSessionExpired (req, resolve, reject) {
```
Comment (davidbarna) [inline src/lib/resource/interceptor/refresh.es]: Please add comments.

File: src/lib/resource/interceptor/manager.es [hunk 2]
```diff
@@ -0,0 +1,148 @@
+/**
```
Comment (davidbarna) [inline src/lib/resource/interceptor/manager.es]: You should be using the one moved to syn-core right?
I suppose these changes are pending... Please answer.

File: src/lib/resource/interceptor/refresh.es (hunk 1 above)
Comment (jcger) [inline src/lib/resource/interceptor/refresh.es]: This is outdated, I moved it to syn-core and I commented it there

File: src/lib/resource/interceptor/manager.es (hunk 2 above)
Comment (jcger) [inline src/lib/resource/interceptor/manager.es]: Yes, I moved it, this is an outdated commit

**Comment by jcger** [comment]
### TODO
- [x] UT

**Comment by davidbarna** [comment]
👍

**Pull request merged**